protocols/cosi/packets.go

```diff
@@ -11,6 +11,12 @@ import (
 
 // The main messages used by CoSi
 
+// see https://github.com/dedis/cothority/issues/260
+// 0 - no check at all
+// 1 - check only at root
+// 2 - check at each level of the tree
+var VerifyResponse = 1
```

Comment:
Oh, I see this isn't a constant. There is not other way to handle this then by a global (exported) variable?